app/assets/javascripts/components/admins/pages/interviews/editable_timeslot.jsx

```diff
@@ -57,10 +65,10 @@ class EditableTimeslot extends AltComponent {
         const formattedTime = formatDateTime(this.state.interview.timeslot);
         return (
             <div className="editable-timeslot-container">
-                { this._renderEditButton() }
                 <EditableInput data         = {formattedTime}
                                editable     = {this.state.editable}
-                               handleChange = {this._onInputChange} />
+                               handleChange = {this._onInputChange}
+                               toggleEdit   = {this._toggleEdit }/>
```

Comment:
Remember to change the comments for EditableInput and the prop type validations!